include/evmc/evmc.hpp

```diff
@@ -345,33 +345,39 @@ class Result : private evmc_result
     ///
     /// @param _status_code  The status code.
     /// @param _gas_left     The amount of gas left.
+    /// @param _gas_refund   The amount of refunded gas.
```

Comment:
Does `gas_left` includes the `gas_refund` value?

Reply:
No. It is mentioned somewhere that gas refund is uncapped. I can try to improve the documentation.

This is implementable, but I decided to leave the transaction-specific logic outside of EVMC yet.